torch/testing/_legacy.py

```diff
@@ -96,6 +97,9 @@ def all_types_and(*dtypes):
 def complex_types():
     return _complex_types
 
+def complex_types_and(*dtypes):
```

Comment:
Should this be added in `_legacy`?

Reply:
cc: @pmeier

Reply:
Yes, it is fine to add it here to keep it consistent with the design. We need to remove it from the list above though. I'll leave a comment there.